README.md

```diff
@@ -30,7 +30,7 @@ Community curated plugins for Core-Lightning.
 | [persistent-channels][pers-chans]    | Maintains a number of channels to peers                                                     |
 | [poncho][poncho]                     | Turns CLN into a [hosted channels][blip12] provider                                         |
 | [pruning][pruning]                   | This plugin manages pruning of bitcoind such that it can always sync                        |
-| [python-teos][python-teos]           | The Eye of Satoshi - Lightning Watchtower                                                   |
+| [rust-teos][rust-teos]               | The Eye of Satoshi - Lightning Watchtower (the [watchtower-plugin](https://github.com/talaia-labs/rust-teos/tree/master/watchtower-plugin) subdirectory)  |
```

Comment:
Just remove this, the watchtower plugin is already in the list. Also rust-teos isn't a plugin.

https://github.com/lightningd/plugins/pull/478/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R39

Reply:
Also I would change line 39 from 

`[watchtower][watchtower-client]` to `[watchtower-client][watchtower-client]`